DESCRIPTION

```diff
@@ -16,7 +16,8 @@ Imports:
     digest,
     crayon
 Suggests:
-    devtools
+    devtools,
+    XML
```

Comment:
In fact, since you're just using it for escaping, I'd prefer no external dep, and just write an R function to do the escaping of `<`, `>` and `&`.